CHANGES.md

```diff
@@ -9,7 +9,9 @@
   * `ThresholdOptmizer` is now a scikit-learn meta-estimator, and accepts
     an estimator through the `estimator` parameter. To use a pre-fitted
     estimator, pass `prefit=True`.
-* Rename arguments of `create_group_metric_set()` to match the dashboard
+* Change the arguments to `create_group_metric_set()`, so that it requires
+  dictionaries for the predictions and sensitive features. This is a
```

Comment:
Please flag that we are changing the name to `_create_group_metric_set` until the API is a bit more stable.

Reply:
Ooops. Thanks for the reminder. Fixed.